e2e/fixtures/partial-build/src/pages/_root.tsx

```diff
@@ -0,0 +1,18 @@
+import { ReactNode } from 'react';
+
+export default function Root({ children }: { children: ReactNode }) {
+  return (
+    <html>
+      <head>
+        <title>Waku</title>
+      </head>
+      <body>{children}</body>
+    </html>
+  );
+}
+
+export const getConfig = async () => {
+  return {
+    render: 'static',
```

Comment:
this is the default, right?

I wonder if we need `_root.tsx` for partial-build fixture. Is there any reason you created this file?

Reply:
I created this just so that we had a test case that used the _root.tsx file.

If the presence of _root.tsx broke the build for any reason, having this here should catch that.

I'll update the description tho!